### PR TITLE
EUI-2017 Navigate in table using keyboard

### PR DESCRIPTION
**Pull request by RiteshHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-2017

### Change description ###
For Event History Table added keyboard accessibility and instruct screen reader user, event details content get changed by pressing Enter button on date column.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by adnanakgun — DISMISSED** [review]
Great job!

File: src/shared/components/palette/history/event-log/event-log-table.component.ts [hunk 1]
```diff
@@ -53,4 +53,12 @@ export class EventLogTableComponent implements OnInit {
   caseHistoryClicked(eventId: string) {
     this.onCaseHistory.emit(eventId);
   }
+
+  ariaLabelText(event: CaseViewEvent): string {
```
Comment (phillip-whitaker) [inline src/shared/components/palette/history/event-log/event-log-table.component.ts:57]: My only comment, but I do not feel greatly passionate about it is... most methods we have like this are name getAriaLabel or buildAriaLabel. So a bit more descriptive about what it does.

Comment (RiteshHMCTS) [reply src/shared/components/palette/history/event-log/event-log-table.component.ts]: Changed the method name to getAriaLabel

**Review by phillip-whitaker — APPROVED** [review]

**Review by adnanakgun — DISMISSED** [review]

**Review by phillip-whitaker — DISMISSED** [review]

**Review by kuda1992 — CHANGES_REQUESTED** [review]

File: src/shared/components/palette/history/event-log/event-log-table.component.ts [hunk 2]
```diff
@@ -53,4 +54,21 @@ export class EventLogTableComponent implements OnInit {
   caseHistoryClicked(eventId: string) {
     this.onCaseHistory.emit(eventId);
   }
+
+  getAriaLabelforColumn(event: CaseViewEvent): string {
```
[the remark this answers is not in the document]
Comment (RiteshHMCTS) [reply src/shared/components/palette/history/event-log/event-log-table.component.ts]: updated

**Review by kuda1992 — APPROVED** [review]
all good

**Review by phillip-whitaker — APPROVED** [review]

**Pull request merged**